From ac6a64496fc99a5cc05ebe4dd7d1b88f60041009 Mon Sep 17 00:00:00 2001
From: Lianbo Jiang <lijiang@redhat.com>
Date: Thu, 10 Dec 2020 11:21:21 +0100
Subject: [PATCH] Missing comma in the dmi_slot_type()

In the initialization of "type", a suspicious concatenated string
"PCI Express Mini 76-pinPCI Express 4 SFF-8639 (U.2)" is produced
due to a missing comma between lines.

[JD: Fixed a similar bug in dmi_processor_characteristics.]

Fixes: b289de9deb32 ("Add new processor characteristics bits from SMBIOS spec 3.4.0")
Fixes: c54348130a2a ("dmidecode: Add new system slot types from SMBIOS spec 3.4.0")
Signed-off-by: Lianbo Jiang <lijiang@redhat.com>
Signed-off-by: Jean Delvare <jdelvare@suse.de>
Signed-off-by: Coiby Xu <coxu@redhat.com>
---
 dmidecode.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/dmidecode.c b/dmidecode.c
index ac96395..4c98553 100644
--- a/dmidecode.c
+++ b/dmidecode.c
@@ -1389,7 +1389,7 @@ static void dmi_processor_characteristics(const char *attr, u16 code)
 		"Execute Protection",
 		"Enhanced Virtualization",
 		"Power/Performance Control",
-		"128-bit Capable"
+		"128-bit Capable",
 		"Arm64 SoC ID" /* 9 */
 	};
 
@@ -1939,7 +1939,7 @@ static const char *dmi_slot_type(u8 code)
 		"PCI Express 3 SFF-8639 (U.2)",
 		"PCI Express Mini 52-pin with bottom-side keep-outs",
 		"PCI Express Mini 52-pin without bottom-side keep-outs",
-		"PCI Express Mini 76-pin"
+		"PCI Express Mini 76-pin",
 		"PCI Express 4 SFF-8639 (U.2)",
 		"PCI Express 5 SFF-8639 (U.2)",
 		"OCP NIC 3.0 Small Form Factor (SFF)",
-- 
2.31.1

